README.md

```diff
@@ -1354,6 +1354,7 @@ _Related: [Monitoring](#monitoring)_
 _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 - [Calagator](https://calagator.org/) - Event aggregator. ([Source Code](https://github.com/calagator/calagator)) `MIT` `Ruby`
+- [ClearFlask](https://clearflask.com) - Community-feedback tool for managing incoming feedback and prioritizing a public roadmap. Alternative to Canny, UserVoice, Upvoty. ([Demo](https://product.clearflask.com), [Source Code](https://github.com/clearflask/clearflask)) `AGPL-3.0` `TS` `Java`
```

Comment:
- https://github.com/clearflask/clearflask/blob/master/INSTALLATION.md#deploy-dependencies -> is it possible to deploy it without the AWS stuff? Else I don't think we can add it to the list

Reply:
Yes it is. The Docker deploy option deploys all dependencies without AWS. 

https://github.com/clearflask/clearflask/blob/master/clearflask-release/src/main/docker/compose/docker-compose.self-host.yml

Reply:
:+1: it looks like Docker is a hard dependency for installation. Please change `TS` `Java`
 to `Docker\